### ISSUE TITLE
Get-myTaskPath

### ISSUE THREAD

**Issue by thecliguy** [jdhitsolutions/MyTasks#36]
# Issue

## PSVersion Table
```
Name                           Value
----                           -----
PSVersion                      5.1.16299.461
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.16299.461
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Since there is a cmdlet to set the `myTaskPath` (`Set-myTaskPath`), it would be useful to have a complementary cmdlet to get the `myTaskPath` path (`Get-myTaskPath`).

**Comment by jdhitsolutions** [comment]
I always assumed it would be easy enough to use Get-Variable., but I guess this would be nicer.

**Comment by thecliguy** [comment]
Yeah the only problem with `Get-Variable` is that you have to know what the name of the variable is and if you don't know then the only way to be absolutely certain is to read the code. Not terribly convenient.

**Comment by jdhitsolutions** [comment]
I agree. New version coming.

**Comment by jdhitsolutions** [comment]
This has been addressed in v2.0.0 which is now in the PowerShell Gallery.
